src/sentry/web/frontend/organization_settings.py

```diff
@@ -162,6 +165,9 @@ def handle(self, request, organization):
             messages.add_message(request, messages.SUCCESS,
                 _('Changes to your organization were saved.'))
 
+            if any([scrubbing_field in form.cleaned_data for scrubbing_field in data_scrubbing_options]):
```

Comment:
Optimize by using a generator.

So:

```python
any((scrubbing_field in form.cleaned_data for scrubbing_field in data_scrubbing_option))
```

The difference is `()` vs `[]`, then it bails out early.